other/block_updates_deletes.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
       Service resource that contains the label `protected=true` unless by
       a cluster-admin.
 spec:
-  validationFailureAction: enforce
+  validationFailureAction: audit
```

Comment:
This one should remain in `enforce` mode because it's pointless to have it in `audit` mode.

Reply:
It seemed to me that other `background: false` validate policies (e.g. block-pod-exec-by-pod-label.yaml) are also useless in `audit` mode.  What do you think?

Reply:
The exec policies should probably be in `enforce` mode, but having those in background mode would help users get a sense if a Pod was exec'd into to assess the value of changing such a policy to `enforce`.

Reply:
The block-pod-exec-* policy uses `request.operation`, so it can't be `background: true`. Should I still leave it as `audit` in this case?

block_updates_deletes.yaml and block-pod-exec-* are very similar rules, so it seems strange that these rules have different values for `validationFailureAction`.

Reply:
Any of the five block policies which use the CONNECT feature probably need to have `background: false`.

Reply:
Sorry, my point was not clear.

Those pod-exec-* policies are already `background: false` and I have not changed `background`.

Those pod-exec-* policies have `validationFailureAction: audit` which I have not changed also. On the other hand, block_updates_deletes.yaml has `validationFailureAction: enforce`, and I changed it to `validationFailureAction: audit` because I think pod-exec-* and block_updates are same kind of rule.

If we leave `validationFailureAction: enforce` of block_updates as it is, then I think other policies that require `background: false` because of `requests.operation`, such as block-pod-exec-*, should also have `validationFailureAction: enforce`.

Reply:
Ok, let's standardize on policies that validate for non-CREATE operations (UPDATE, CONNECT, DELETE) should be set `validationFailureAction: enforce`. Background mode should be set accordingly to `false`.

Reply:
Thank you for the comment. I changed policies that validate for non-CREATE operations (UPDATE, CONNECT, DELETE) to have validationFailureAction: enforce.